src/Controller/MediaController.php

```diff
@@ -110,7 +110,7 @@ public function viewAction($id, $format = MediaProviderInterface::FORMAT_REFEREN
      *
      * @return Response
      *
-     * @deprecated since 3.x, to be removed in 4.0.
+     * @deprecated since 3.12, to be removed in 4.0.
```

Comment:
Check on your own if you think this is a mistake.

Reply:
It was deprecated since 3.12, see #1393